src/components/skip-link/index.md.njk

```diff
@@ -15,7 +15,9 @@ Use the skip link component to help keyboard-only users skip to the main content
 
 ## When to use this component
 
-All GOV.UK pages must include a skip link in the header.
+All GOV.UK pages must include a skip link, placed on its own at the top of the `<body>` tag, and outside of the `<header>' tag.
+
+Do not put the skip link inside the `<header>` tag. Accessibility testing tools that expect all elements to be inside a landmark will flag it as a failure.
```

Comment:
Elements or components?

Reply:
I'd go with 'elements' as that's the language that would be used by the automated tools.

I do think this sentence is a bit slightly ambiguous and potentially confusing. The way I naturally read it is:

> Do not put the skip link inside the `<header>` tag, _because if you did_ accessibility testing tools that expect all elements to be inside a landmark _would_ flag it as a failure.

That's incorrect – I think what we're trying to say is:

> Do not put the skip link inside the `<header>` tag, _even though_ accessibility testing tools that expect all elements to be inside a landmark will flag it as a failure.

However, there are also other ways that users might try and 'fix' the 'all elements to be inside a landmark' warning, so I don't think it makes sense to tie the two together.

How about something like:

> All GOV.UK pages must include a skip link. Place the skip link immediately after the opening `<body>` tag.
> 
> Some automated accessibility testing tools may warn that the skip link element is not contained within a landmark. This does not apply to skip links, and so the warning can be ignored.

We could follow with 'Do not wrap the skip link in a nav region, or move it inside the header.' if we want to be really explicit about that?

Reply:
That makes sense! Thanks. Yeah, being really explicit sounds good, if it could save users from resorting to workarounds.